packages/sampler/stories/default/ViewManager.js

```diff
@@ -70,13 +79,15 @@ const ViewManagerLayout = (props) => {
 				end={end}
 				index={selected}
 				start={start}
+				enteringProp={enteringProp}
+				enteringDelay={enteringDelay}
 				style={{height: ri.scale(42 * (end - start + 1)), overflow: 'hidden'}}
 				{...rest}
 			>
 				{views.map((view, i) => (
-					<div className={css.box} key={i}>
+					<ChildrenDiv className={css.box} enteringProp={enteringProp} key={i} >
```

Comment:
Let's update this code to use another way (e.g. passing 'args') rather than using `enteringProp` directly since it could lead to a misunderstanding that an app developer should pass `enteringProp` to children components.

Reply:
fixed it.